docs/data/opensearch.yml

```diff
@@ -17,6 +17,7 @@
 ################################################################################
 
 version: 2.0.0-SNAPSHOT
```

Comment:
Is this version correct?
In the `v1.2` branch the version is `1.2-SNAPSHOT`
https://github.com/apache/flink-connector-opensearch/blob/419632cc58cb81cbba47a9094576cb1a6a95978d/pom.xml#L31